cmd/wasmcli/main.go

```diff
@@ -43,7 +44,7 @@ func main() {
 		WithInput(os.Stdin).
 		WithAccountRetriever(authtypes.NewAccountRetriever(encodingConfig.Marshaler)).
 		WithBroadcastMode(flags.BroadcastBlock).
-		WithHomeDir(app.DefaultNodeHome)
+		WithHomeDir(app.DefaultCLIHome)
```

Comment:
Very good call.

Reply:
Is this a bug/issue in master as well?

Reply:
No, that was me migrating the client context 🙈